app/models/user.rb

```diff
@@ -53,4 +50,15 @@ def update_password_and_token(password)
     self.token = nil
     self.save
   end
+
+  def connect_with_invitors
+    invites = Invite.where(email: self.email)
+    if invites.exists?
+      invitor_ids = invites.map(&:user_id).uniq
```

Comment:
I wouldn't say this is a fix necessarily, just a suggestion of another way to do it. Sometimes pluck is worth it, and sometimes not. I just wanted you to be aware of the option. If you didn't get a chance to read that link, I highly recommend reading the whole thing. I learned a lot from it.